www/css/index.css

```diff
@@ -85,3 +85,7 @@ body {
 .ac-drop-down > li {
   opacity: inherit;
 }
+
+#textfield-Description {
+  font-family: Arial
```

Comment:
When selecting from a font-family, always have `serif` (or `sans-serif` in this case) as a back up.
w3schools example here: http://www.w3schools.com/css/css_font.asp
mdn for more information: https://developer.mozilla.org/en-US/docs/Web/CSS/font